dask/dataframe/tests/test_rolling.py

```diff
@@ -278,25 +278,25 @@ def test_rolling_agg():
     expected = pd.concat([a_mean, a_std, b_mean, b_std], axis=1)
     expected.columns = pd.MultiIndex.from_product([['A', 'B'], ['mean',
                                                                 'std']])
-    tm.assert_frame_equal(result, expected)
+    assert_eq(result, expected)
 
     result = r.aggregate({'A': np.mean, 'B': np.std}).compute()
```

Comment:
Ideally we would drop the `.compute()` method here when we use `assert_eq`.  By giving `assert_eq` a pre-computed result it can run a bunch of checks on the dask object itself.